pkg/ebpf/c/vmlinux.h

```diff
@@ -582,9 +582,11 @@ struct icmp6hdr {
 
 struct linux_binprm {
 	struct file *file;
+	struct file *interpreter;
```

Comment:
are you using these added filelds?

Reply:
No, it is a left over of something I did before the current version.
Nice remark!
I will fix it soon.